bin/demultiplexingTiming.sh

```diff
@@ -172,54 +173,49 @@ do
-    if [[ -z "${timeStamp}" ]]
+    if [ -e "${SCR_ROOT_DIR}/logs/${sequenceRun}/${run}.demultiplexing.finished" ]
     then
-        echo "timestamp is niet leeg!"
-        if [ -e "${SCR_ROOT_DIR}/logs/${sequenceRun}/run0"[1-9]".demultiplexing.finished" ]
+        echo ".finished bestaat"
+        declare -a sampleSheetColumnNames=()
+        declare -A sampleSheetColumnOffsets=()
+        declare -a projects=()
+        
+        IFS="," sampleSheetColumnNames=($(head -1 "${sampleSheet}"))
+        
+        for (( offset = 0 ; offset < ${#sampleSheetColumnNames[@]:-0} ; offset++ ))
+        do
+            sampleSheetColumnOffsets["${sampleSheetColumnNames[${offset}]}"]="${offset}"
+        done
+
+        if [[ ! -z "${sampleSheetColumnOffsets['project']+isset}" ]] 
         then
-            echo ".finished bestaat"
-            declare -a sampleSheetColumnNames=()
-            declare -A sampleSheetColumnOffsets=()
-            declare -a projects=()
-            
-            IFS="," sampleSheetColumnNames=($(head -1 "${sampleSheet}"))
-            
-            for (( offset = 0 ; offset < ${#sampleSheetColumnNames[@]:-0} ; offset++ ))
-            do
-                sampleSheetColumnOffsets["${sampleSheetColumnNames[${offset}]}"]="${offset}"
-            done
-
-            if [[ ! -z "${sampleSheetColumnOffsets['project']+isset}" ]] 
-            then
-                projectFieldIndex=$((${sampleSheetColumnOffsets['project']} + 1))
-                IFS=$'\n' projects=($(tail -n +2 "${sampleSheet}" | cut -d "," -f "${projectFieldIndex}" | sort | uniq ))
-            fi
-
-            for project in "${projects[@]}"
-            do
-                echo "${project}"
-                projectStampFile="${timeStampDir}/${project}.csv"
-                if [[ -e "${projectStampFile}" ]]
-                then
-                    echo "${projectStampFile} already exists"
-                else
-                    printf "${sequenceRun}" >> "${projectStampFile}" 
-                fi
-                touch "${SCR_ROOT_DIR}/logs/${sequenceRun}/${SCRIPT_NAME}.finished"
-                echo "Demultiplexing is finished"
-            done
-        else
-            echo ".started file is NOT older than 1 hours. Demultiplexing is running"
-
+            projectFieldIndex=$((${sampleSheetColumnOffsets['project']} + 1))
+            IFS=$'\n' projects=($(tail -n +2 "${sampleSheet}" | cut -d "," -f "${projectFieldIndex}" | sort | uniq ))
         fi
 
+        for project in "${projects[@]}"
+        do
+            echo "${project}"
+            projectStampFile="${timeStampDir}/${project}.csv"
+            if [[ -e "${projectStampFile}" ]]
+            then
+                echo "${projectStampFile} already exists"
+            else
+                echo "${sequenceRun}" >> "${projectStampFile}"
+            fi
+            touch "${SCR_ROOT_DIR}/logs/${sequenceRun}/${run}.${SCRIPT_NAME}.finished"
+            echo "Demultiplexing is finished"
```

Comment:
in cronjobs an echo command will be lost in translation, you should put it in a log file instead